src/Panel.vue

```diff
@@ -201,10 +201,19 @@
         return this.cardType === 'bg-light' || this.cardType === 'bg-white' || this.cardType === 'bg-warning';
       },
       headerContent () {
-        return md.render(this.header);
+        if (this.isSeamless) {
+            return this.caretHtml + ' ' + this.inlineRenderedHeader;
+        }
+        return this.renderedHeader;
```

Comment:
As discussed, maybe everything can just use `inline` then, since for non-seamless, it seems to not make a difference, there is no reason to do different rendering ways for different type of panels.